definitions/variable/tag_final_energy.yaml

```
@@ -0,0 +1,33 @@
+- Final Energy Carrier:
+    - Solids:
+        description: Solid fuels
+    - Solids|Coal:
+        description: Anthracite, coking coal, lignite, peat
+    - Solids|Biomass:
+        description: Solid biomass
+    - Liquids:
+        description: Liquid fuels
+    - Liquids|Oil:
+        description: Oil products refined from crude oil e.g. gasoline, diesel, gas oil, fuel oil
+    - Liquids|Biomass:
+        description: Liquid biomass, biofuels
+    - Liquids|Synfuel:
+        description: Synthetic fuels
```

Consistentcy with electricity fuels above
```suggestion
    - Liquids:
        description: Liquid fuels
    - Liquids|Fossil:
        description: Oil products refined from crude oil e.g. gasoline, diesel, gas oil, fuel oil
    - Liquids|Bio:
        description: Liquid biomass, biofuels
    - Liquids|Synfuel:
        description: Synthetic fuels
```